Sources/BraintreeCore/Public/BraintreeCore/BTPostalAddress.h

```diff
@@ -10,6 +10,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  Optional. Recipient name for shipping address.
 */
+// Maybe `fullName` (see graphql) with a note: Full name, (eg. recipient when shipping).
```

Comment:
I assume if we change to `fullName` we want to make this change across all SDKs (which still use `recipientName`).

Also, the [GraphQL docs ](https://graphql.braintreepayments.com/reference/#object--address) also don't use `PostalAddress` but rather `Address`. Is this also a change we want to make?

Reply:
@demerino

Reply:
I think we should punt on this, unless we think this is particularly confusing or problematic for merchants. There are probably quite a few fields where the SDK names don't match GraphQL, but I don't think that should hold up the release.

Maybe consistency with GraphQL can be an objective of the v6 release?

Reply:
Confirmed: punting